examples/suppressions.py

```
@@ -0,0 +1,22 @@
+from typing import Optional
+
+import mailtrap as mt
+from mailtrap.models.suppressions import Suppression
+
+API_TOKEN = "YOU_API_TOKEN"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "API_TOKEN"

(S105)